packages/swayze/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.0
+
+- Removes `id` property from `UserSelectionModel`.
```

Comment:
This is technically a breaking change, although I doubt there is someone using this `id`. 
I think for the time being we should mark this as deprecated and remove it from the API in a future major version (i.e., `2.0.0`). What do you think? 

Another option would be to release `2.0.0` already, but I think that would be too hasty. Specially since we have other PRs that would probably make sense to bundle in a new version release

Reply:
I think you're right. Let me revert the changes and mark it as deprecated, and then we remove it on version 2.

Reply:
Thanks 😄